static/src/stylesheets/module/content/_live-blog.scss

```diff
@@ -478,12 +478,11 @@ $block-padding-right: $gs-gutter;
 .dropdown--live-feed {
     border-top: 0;
 
+    $content-gutter: $gs-gutter/2;
+
     @include mq($until: desktop) {
-        margin-right: $gs-gutter*-1;
-        margin-left: $gs-gutter*-1;
-        padding-right: $gs-gutter;
-        padding-left: $gs-gutter;
-        padding-top: $gs-baseline*4;
+        margin-right: ($content-gutter)*-1;
```

Comment:
I've used this variable further down as well. If someone changes it, they won't accidentally forget to change it there too…

Will add some white space